components/core/src/clp/GlobalMetadataDBConfig.cpp

```diff
@@ -1,110 +1,97 @@
 #include "GlobalMetadataDBConfig.hpp"
 
+#include <cstdlib>
+#include <stdexcept>
+
 #include <fmt/core.h>
-#include <yaml-cpp/yaml.h>
 
 using std::exception;
 using std::invalid_argument;
 using std::string;
 
-static exception get_yaml_missing_key_exception(string const& key_name) {
-    throw invalid_argument(fmt::format("Missing key '{}'", key_name));
+namespace clp {
+auto GlobalMetadataDBConfig::add_command_line_options(
+        boost::program_options::options_description& options_description
+) -> void {
+    options_description.add_options()
```

Comment:
instead of adding to the general options, we can create a DB config specific subsection like the Input / Output options